requirements.txt

```
@@ -1,4 +1,5 @@
 fiona
+geoviews
 geopandas
```

This file is sorted alphabetically

```suggestion
geopandas
geoviews
```